docs/guides/compose.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Compose
+---
```

Comment:
Need a note about using `.tailcallrc.graphql` and `.tailcallrc.schema.json` with json and yml.

Reply:
@neo773 need to add some context about these files as well. The files help in autocomplete. Would be great to showcase that feature here.